Upgrade tech-docs to latest version for accessibility changes

klssmith opened the pull request:
Version `2.2.2` of `govuk_tech_docs` brings in various accessibility
changes. We don't need to make any changes to the code for these to take
effect.

The accessibility statement has been updated to remove references to the
tech docs template not being accessible. I've also changed the file
extension of the accessibility statement since it doesn't contain any
Ruby code - this has no effect on its appearance.

[Pivotal story](https://www.pivotaltracker.com/story/show/177362717)

klssmith commented:
@karlchillmaid can you check the updates to the accessibility statement please?

karlchillmaid commented on source/accessibility.html.md:
```diff
@@ -23,10 +23,9 @@ The text should be clear and simple to understand.
 ## How accessible this website is
 
-We know some parts of this website are not fully accessible for the following reasons:
+We know some parts of this website are not fully accessible for the following reason:
 
 - some pages have adjacent links that go to the same URL
```
```suggestion
We know this website is not fully accessible because some pages have adjacent links that go to the same URL.
```

klssmith replied:
Thanks Karl, I've amended the original commit to make that change.

karlchillmaid reviewed (approved):
Apart from the small change I made the updates to the accessibility statement look good.

tombye commented:
The only issue that, I think, still needs addressing from the google doc is putting row headings into some tables.

I think the following types of table should have rows where the first cell is a `th` with `scope=row`:
1. tables that list info’ about JSON fields, ie. https://docs.notifications.service.gov.uk/java.html#delivery-receipts
2. tables listing error codes, ie. https://docs.notifications.service.gov.uk/java.html#error-codes
3. tables listing limits, ie. https://docs.notifications.service.gov.uk/java.html#daily-limits

It does give that first cell bold text, which I think is OK for 1. and 2. as it's quite subtle. Number 3. looks a bit odd as it makes the cell text match the column headings:

<img width="822" alt="status_table_with_headings" src="https://user-images.githubusercontent.com/87140/112507802-e27e6380-8d86-11eb-907f-2c2c752bbbe4.png">

We could hide the column headings for visual users, like we do in places on the admin app. Their information is in the `h3` before the table so I don't think you lose anything:

<img width="798" alt="status_table_no_headings" src="https://user-images.githubusercontent.com/87140/112508180-3be69280-8d87-11eb-8a4a-a7aefe085a9b.png">

klssmith commented:
I've now added markup to the headers in 711f929 also replaced the single column tables for fake email address and phone numbers with bullet points:
<img width="400" alt="Screenshot 2021-03-26 at 14 53 19" src="https://user-images.githubusercontent.com/12881990/112662620-cdb9d280-8e4f-11eb-84a3-647fcdcb830f.png">

klssmith commented:
To try these changes out locally, replace this line https://github.com/alphagov/notifications-tech-docs/blob/75dab972d5c23396b9f53313af0ec2d487a3bfc5/app/external_doc.rb#L5 with 
```
url = "https://raw.githubusercontent.com/#{repository}/tables/DOCUMENTATION.md"
```

tombye reviewed (approved):
Looks good to me 👍
